pages/devel-version.html

```diff
@@ -200,7 +200,7 @@ <h5>Hardware and Software Requirements</h5>
                     <tbody >
                     <tr>
 
-                        <td>openSUSE Leap 15.4</td>
+                        <td>openSUSE Leap 15.5</td>
```

Comment:
Not critical, but as you removed Ubuntu 18.04 from stable, maybe you want to remove it here as well, at line 252?

Reply:
Removed, good catch.

Reply:
Do you mean from the list of repos? because we removed it

Reply:
@deneb-alpha From Stable yes, but not from Devel. But @raulillo82 did it now :-)